site/src/components/pages/ZkSharding/Hero/Hero.module.scss

```diff
@@ -23,37 +24,18 @@
 
   @include mobile {
     padding-left: size($spacing12);
+    padding-top: size($spacing24 - 8);
```

Comment:
![Снимок экрана 2023-12-15 в 16 00 43](https://github.com/NilFoundation/nil.foundation/assets/63141840/c4cecfbe-2e23-486a-9f3b-8e16e90a0bb0)

Reply:
Well, i mean that `$spacing24 - 8` is equal to 24 - 8, isn't it right?

Reply:
Yes, it is equal to 24 - 8

Judging by the design, the padding should be 24 pixels, but due to the difference in font drawing (Figma and browser), we reduce the paddings to leave dependence on the $spacing24 variable

Reply:
Does this issue relate to the particular font (in our case Helvetica Neue LT Pro) or it is something with drawing font in general? May i ask you to elaborate on this a bit more?

Reply:
in general, Figma draws the font a little differently, but in this case it is related to a certain font. Throughout the project, I adjusted the margins to the font so that it was pixel perfect